ci/docker/conda-python-spark.dockerfile

```
@@ -23,11 +23,16 @@ FROM ${repo}:${arch}-conda-python-${python}
 ARG jdk=8
 ARG maven=3.5
 
+ARG numpy=latest
+COPY ci/scripts/install_numpy.sh /arrow/ci/scripts/
+
 RUN mamba install -q -y \
         openjdk=${jdk} \
         maven=${maven} \
         pandas && \
     mamba clean --all
+RUN mamba uninstall -q -y numpy && \
```

How about concatenating these commands to reduce intermediate Docker image?

```suggestion
    mamba clean --all && \
    mamba uninstall -q -y numpy && \
```